backend/BackendMIIT/src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
 spring.application.name=BackendMIIT
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:postgresql://localhost:5432/MiitInside
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+spring.datasource.driver-class-name=org.postgresql.Driver
```

Comment:
Add the empty line in the end of the file :D

Reply:
Added